Update 4.60.0 changelog (#14151)
hashicorp/terraform-provider-google: CHANGELOG.md

## 4.60.0 (Unreleased)

FEATURES:
* **New Data Source:** `google_compute_region_instance_template` ([#14132](https://github.com/hashicorp/terraform-provider-google/pull/14132))

> * **New Resource:** `apigeeKeystoresAliasesKeyCertFile` ([#14130](https://github.com/hashicorp/terraform-provider-google/pull/14130))

Reviewer: The resource name is incorrectly formatted here

Author: Changed.

* **New Resource:** `google_compute_region_instance_template` ([#14103](https://github.com/hashicorp/terraform-provider-google/pull/14103))

IMPROVEMENTS:
* compute: added 3000 as allowable value for `disk_size_gb` for SCRATCH disks in `google_compute_instance_template` ([#14061](https://github.com/hashicorp/terraform-provider-google/pull/14061))

> * compute: added missing fields for `google_compute_(?.*_)address` datasource ([#14078](https://github.com/hashicorp/terraform-provider-google/pull/14078))

Reviewer: Which fields, and which resource?

Author: Changed

* compute: added `network_firewall_policy_enforcement_order` field to `google_compute_network` resource ([#14111](https://github.com/hashicorp/terraform-provider-google/pull/14111))

> * compute: added output-only attribute `self_link_unique` id for `google_compute_instance_template` to point to the unique id of the resource (instead of its name) ([#14128](https://github.com/hashicorp/terraform-provider-google/pull/14128))

Reviewer: ```suggestion
* compute: added output-only attribute `self_link_unique` for `google_compute_instance_template` to point to the unique id of the resource instead of its name ([#14128](https://github.com/hashicorp/terraform-provider-google/pull/14128))
```

Author: Changed

> * compute: supported region instance template in`source_instance_template` field of `google_compute_instance_from_template` resource ([#14103](https://github.com/hashicorp/terraform-provider-google/pull/14103))

Reviewer: ```suggestion
* compute: added support for region instance template in`source_instance_template` field of `google_compute_instance_from_template` resource ([#14103](https://github.com/hashicorp/terraform-provider-google/pull/14103))
```

Author: Changed

* container: added `stack_type` field to `google_container_cluster` resource ([#14079](https://github.com/hashicorp/terraform-provider-google/pull/14079))

> * container: added support for `advanced_machine_features.threads_per_core` ([#14106](https://github.com/hashicorp/terraform-provider-google/pull/14106))

Reviewer: In what resource?

Author: Changed

* networkservice: updated the max number of `host_rule` on `google_network_services_edge_cache_service` ([#14112](https://github.com/hashicorp/terraform-provider-google/pull/14112))
* sql: added support of single-database-recovery for SQL Server PITR with `database_names` attribute to `google_sql_instance` ([#14088](https://github.com/hashicorp/terraform-provider-google/pull/14088))

BUG FIXES:
* cloudrun: fixed race condition when polling for status during an update of a `google_cloud_run_service` ([#14087](https://github.com/hashicorp/terraform-provider-google/pull/14087))
* cloudsql: fixed the error in any subsequent apply on `google_sql_user` after its `google_sql_database_instance` is deleted ([#14098](https://github.com/hashicorp/terraform-provider-google/pull/14098))

> * datacatalog: fixed `google_data_catalog_tag` only allowing 10 tags.. increased to 1000 ([#14077](https://github.com/hashicorp/terraform-provider-google/pull/14077))

Reviewer: ```suggestion
* datacatalog: fixed `google_data_catalog_tag` only allowing 10 tags. increased to 1000 ([#14077](https://github.com/hashicorp/terraform-provider-google/pull/14077))
```

Author: Changed
